qiskit/utils/deprecation.py

```diff
@@ -26,7 +26,7 @@ def deprecate_func(
     since: str,
     additional_msg: str | None = None,
     pending: bool = False,
-    package_name: str = "qiskit",
+    package_name: str = "Qiskit",
```

Comment:
The docstring for this argument indicates that it ought to be the PyPI package name. Assuming we want to make this change, we ought to update the docstring (and probably the arg name as well, e.g. `package_description`).

Do we use `deprecate_func` for packages other than Qiskit at this point? If not, perhaps we should just remove the argument altogether and hard-code `"Qiskit"`.

Reply:
Updated in 5dd8262, along with a test that setting `package_name` works as expected